app/tempCardPage.tsx

```diff
@@ -0,0 +1,16 @@
+import { Text, View, StyleSheet, ScrollView, Dimensions } from 'react-native';
+import { Link } from 'expo-router';
+import MultipleChoice from '../components/ui/FlashCards/MultipleChoice';
+import MainButton from '../components/ui/Buttons/MainButton';
+// import { ScrollView } from 'react-native-gesture-handler';
+
+
+
+export default function TempCardPage () {
+
+    return (
+        <View>
+        <MultipleChoice />
```

Comment:
Make sure your using prettier on save, and that prettier is looking at the .prettierrc file in the project

Reply:
👍 kind of forgot about this file